### DISCUSSION TITLE
FYI - Compositional Linear Algebra (CoLA) Library

### DISCUSSION THREAD

**Discussion by adam-hartshorne** [pnkraemer/probdiffeq#733] [Ideas]
I don't know if you have seen this new library, but it seems like it could provide some useful efficient Linear Operators for probabilistic ode solvers.

CoLA is a framework for scalable linear algebra, automatically exploiting the structure often found in machine learning problems and beyond. CoLA supports both PyTorch and JAX.

https://github.com/wilson-labs/cola

**Comment by pnkraemer** [comment]
Hi! I am converting this issue to a discussion since there is no concrete issue to resolve. Feel free to continue commenting there!

**Comment by adam-hartshorne** [comment]
Notice you are doing a lot of dev at the moment. Just wondering if you thought anymore about this, I personally have found this library quite useful, provided a good speed-up / better memory usage.
